src/components/organisation/searchOrg.stories.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+
+import SearchOrg from './searchOrg';
+import { withKnobs, object } from '@storybook/addon-knobs/react';
+
+// attaches component to the story book
+export default {
+	component: SearchOrg,
+	title: 'SearchOrg',
+	decorators: [withKnobs], // use to display state in storybook
+	excludeStories: /.*Data$/, // export ending with Data wont be treated as a story
+};
+
+export const initialStateData = {
+	organisations: [
+		{
+			org_id: 1,
+			org_name: 'E-Cell NIT Raipur',
+			org_desc: 'Leader Beyond Borders, Who do everything they ca d',
+		},
+		{
+			org_id: 2,
+			org_name: 'E-Cell NIT Raipur',
+			org_desc: 'Leader Beyond Borders',
+		},
+		{
+			org_id: 3,
+			org_name: 'E-Cell NIT Raipur',
+			org_desc: 'Leader Beyond Borders',
+		},
+		{
+			org_id: 4,
+			org_name: 'E-Cell NIT Raipur',
+			org_desc: 'Leader Beyond Borders',
+		},
+		{
+			org_id: 5,
+			org_name: 'E-Cell NIT Raipur',
+			org_desc: 'Leader Beyond Borders',
+		},
+		{
+			org_id: 6,
+			org_name: 'E-Cell NIT Raipur',
+			org_desc: 'Leader Beyond Borders',
+		},
+		{
+			org_id: 7,
+			org_name: 'E-Cell NIT Raipur',
+			org_desc: 'Leader Beyond Borders',
+		},
+	],
+	loading: false,
+	search: [],
+	search_value: '',
+	search_flag: false,
+};
+
+export const loadingStateData = {
+	...initialStateData,
```

Comment:
there won't be any org in array if loading is true, when loading is done, then state will be filled wilth data, so no need of this line.

Reply:
ok